content/languages/ocaml/recipes.md

```diff
@@ -0,0 +1,34 @@
+---
+title: OCaml recipes for Codewars
+tags: [authoring, recipe, ocaml]
+---
+
+
+## Informative assertion messages
+
+Default assertion messages of OCaml OUnit are usually very bad. OUnit assertons accept two optional, named arguments:
+- `~printer`, used to stringify values presented by assertion messages.
```

Comment:
I recommend the following definition:
`~printer` defines a printer (stringifier) for compared values.

Reply:
For me, your wording sounds more like "stringified values are compared" rather than "stringified values are presented in a message".

But I trust your experience more than my feelings, so I go with your proposal.